tools/docker/tests/test_gw_repeater.sh

```diff
@@ -76,25 +76,25 @@ main() {
     dbg DELAY=$DELAY
 
     status "Start GW (Controller + local Agent)"
-    ${scriptdir}/../run.sh ${OPT} start-controller-agent -d -n ${GW_NAME} -m 00:11:22:33
+    ${scriptdir}/../run.sh ${OPT} start-controller-agent -d -n ${GW_NAME} -m 00:11:22:33 $@
```

Comment:
I think this should be `"$@"`.

In general, i the quotes are not needed, I prefer to use `$*` instead (the two are equivalent when it's not quoted). Since `$*` is less used, it draws attention to the missing quotes.